Newsletter: An engineer's guide to talking to users

ivanagas opened the pull request:
## Changes

An engineer's guide to talking to users. 

Closes #8252

## Checklist

- [x] Words are spelled using American English
- [x] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [x] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [x] Use relative URLs for internal links

## Article checklist

- [x] I've added (at least) 3-5 internal links to this new article
- [x] I've added keywords for this page to the rank tracker in Ahrefs
- [ ] I've checked the preview build of the article
- [x] The date on the article is today's date
- [ ] I've added this to the relevant "Tutorials and guides" docs page (if applicable)

## Useful resources

- [The PostHog style guide](https://posthog.com/handbook/growth/marketing/posthog-style-guide)
- [Full list of tags and categories](https://posthog.com/handbook/growth/marketing/tags-and-categories)
- [List of content components](https://posthog.com/handbook/growth/marketing/components)
- [PostHog SEO best practices](https://posthog.com/handbook/growth/marketing/seo-guide)

jtemperton commented:
Structure works well, and I think we cover a lot of ground. Nothing structural that needs changing, I don't think. It's a good read!

Few things to work on, I reckon...

The intro is a little flat, could do with livening up. And throughout I'd suggest looking at sentence structure. Split some of them up, use a mix of sentence structures, bringing in some short, sharp sentences for emphasis, etc.

I'd also lean way less on Farley throughout. We should be giving our advice and experience and a good variety of this from other sources. Right now there's a lot of Farley. 

The block-quotes from Farley are a bit of a turn-off for me and I think we'd be better, where we can, speaking from our experience of bringing in examples from other companies. The line about Microsoft, for example, works well. I'd lose all the Farley block quotes and use a fun line or two from him, max. And maybe not reference him throughout.

Beyond that the main thing that needs work is tone and pacing in some places. I think it could be a bit shorter and a little zippier. The final section (Iterating beyond user interviews) feels a little long.

ivanagas commented:
Ok, made some edits.

- Rewrote intro
- Heavily edited (shortened) all the other sections
- Removed all but one Farley quote. It's just hard to make such broad generalizations about engineering without an expert to back it up (maybe I shouldn't be doing this though).

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+  - Blog
+---
+
+Talk to users? Isn't that a job for founders, product managers, user researchers, and salespeople? Why do would an engineer ever want or need to do it?
```
maybe to avoid the three ??? something more like

Why would an engineer ever need to talk to users? That's a job for founders, product managers, user researchers, and salespeople. Right? Wrong.

In his book...

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+In his book [*Modern Software Engineering*](https://www.davefarley.net/?p=352), Dave Farley writes that mastering software engineering requires becoming an expert in learning. When learning is prioritized, talking to users becomes a critical part of an engineer's toolkit.
+
+To explain further why this is the case, and to help you get started, we wrote this guide based on what we've learned talking to users as engineers ourselves at PostHog.
```
Maybe more like...

So that begs the question: how should engineers be talking to users? Fret not, we've assembled an essential guide that covers everything you need to know.

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+Engineers' biggest pushback against talking to users is that it is a waste of time. This belief is a symptom of the wrong priorities.
+
+Many engineers place shipping code above all else. Anything that gets in thwe way of this is a waste of time. This is reflected in common metrics measuring engineering success like lead time or deployment frequency. 
```
```suggestion
Many engineers place shipping code above all else. Anything that gets in thwe way of this is, apparently, a waste of time. This is reflected in common metrics for measuring engineering success like lead time or deployment frequency. 
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+Many engineers place shipping code above all else. Anything that gets in thwe way of this is a waste of time. This is reflected in common metrics measuring engineering success like lead time or deployment frequency. 
+
+This creates a disregard for shipping the *right* code: features users value. Studies show this is more common than you might think. For example, [Microsoft](https://ai.stanford.edu/~ronnyk/ExPThinkWeek2009Public.pdf) found that **only one-third** of ideas tested improved metrics they were designed for.
```
```suggestion
This creates a disregard for shipping the right code: features users value. Studies show this is more common than you might think. For example, [Microsoft](https://ai.stanford.edu/~ronnyk/ExPThinkWeek2009Public.pdf) found that only one-third of ideas tested improved metrics they were designed for.
```

jtemperton replied:
the words themselves add the emphasis, we don't need to bold and italicize

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+This creates a disregard for shipping the *right* code: features users value. Studies show this is more common than you might think. For example, [Microsoft](https://ai.stanford.edu/~ronnyk/ExPThinkWeek2009Public.pdf) found that **only one-third** of ideas tested improved metrics they were designed for.
+
+A fix for this is spending more time learning what users want. Nothing beats doing this yourself directly. By doing this, you gain the knowledge to that closes the gap between what users value and what you are building.
```
```suggestion
A fix for this is spending more time learning what users want. And nothing beats doing this yourself. This will also help you close the gap between what users value and what you are building.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+![Don't build the wrong features](https://res.cloudinary.com/dmukukwp6/image/upload/v1713470851/posthog.com/contents/images/newsletter/talk-to-users/features.jpg)
+
+## Finding users to talk with
```
```suggestion
## Finding users to talk to
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+## Finding users to talk with
+
+Just because we convinced you that talking to users isn't a waste of time does not mean we want to waste your time doing so. We realize engineers are busy, but that isn't an excuse to not talk to users. As Farley writes:
```
```suggestion
Just because we convinced you that talking to users isn't a waste of time doesn't mean we want to waste your time doing so. We realize engineers are busy, but that isn't an excuse to not talk to users. As Farley writes:
```

jtemperton replied:
i think we want to use contractions more often than not, stylistically

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+> Modern teams fight with schedule pressure, quality, and maintainability of their designs. They often struggle to identify the ideas that really land with users, and they fail to allow themselves the time to learn about the problem domain, the technology, and the opportunities to get something great into production.
+
+The lucky for you, you don't need a lot of users. This doesn't require a statistically significant sample. A few user interviews can be all you need to clarify the next increment of progress.
```
```suggestion
Lucky for you, you don't need a lot of users. This doesn't require a statistically significant sample. A few user interviews can be all you need to clarify the next increment of progress.
```

jtemperton replied:
what does the 'this' in 'This doesn't require a statistically significant...' refer to?

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+![Feedback for merch code](https://res.cloudinary.com/dmukukwp6/image/upload/v1713470870/posthog.com/contents/images/newsletter/talk-to-users/merch.png)
+
+## What NOT to talk about
```
```suggestion
## What you shouldn't talk about
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+## What NOT to talk about
+
+Ironically enough, information from talking to users comes from **NOT TALKING**. Beyond prioritizing listening, there are some specific areas you should avoid. They include:
```
```suggestion
The best information you'll get from talking to users comes from not talking. Beyond prioritizing listening, there are some specific areas you should avoid. They include:
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+2. **Your solution.** Resist the urge to explain your features and your thinking. You shouldn't use the time to convince them why your solution is the right one.
+
+3. **Their solutions.** Users likely recommendations and feature requests. These often don't solve their underlying problems, so reframe these to dig deeper into the underlying causes.
```
is there a missing word here?

 _Users likely recommendations and feature requests_

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+4. **Macroanalysis**. Focus on an individual's specific use case rather than a company or industry one. Outside research can fill this if necessary.  
+
+## What to talk about?
```
```suggestion
## What you should talk about
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+Our experience talking to users has revealed two main discussion areas. The first is **problem exploration**. This guides future product decisions, like [what to build next](https://newsletter.posthog.com/p/how-we-decide-what-to-build). Focus on concrete situations by asking questions like:
+
+- How are you solving your problem currently?
```
```suggestion
- How are you solving your problem right now?
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+- How much would you pay for a solution to your problem?
+- What is your workflow for solving this problem? Can you talk me through it?
+
+The second is **solution validation**. This works best when they are already using your feature or you have a demo to show them. Focus on impressions and problem areas by asking questions like:
```
```suggestion
The second is **solution validation**. This works best when someone is already using your feature or you have a demo to show them. Focus on impressions and problem areas by asking questions like:
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+The second is **solution validation**. This works best when they are already using your feature or you have a demo to show them. Focus on impressions and problem areas by asking questions like:
+
+- Have you tried using X? If not, why?
```
```suggestion
- Have you tried using X? If not, why not?
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+- How are you using X now? What are your touchpoints and workflows with X?
+- What is confusing?
+
+Dive deeper into interesting workflows and lines of thought. Get their preference on options. Ask about constraints. Finally, take notes and [share them with your team](https://posthog.com/product-engineers/interview-snapshot-guide) so they can benefit too (we use an AI transcription tool and keep a master interview document).
```
what do you mean by 'get their preference on options'?

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+## Iterating beyond user interviews
+
+In *Modern Software Engineering*, Farley outlines 5 behaviors engineers need to become experts in learning: iteration, feedback, incrementalism, experimentation, and empiricism. This shows that the process of learning doesn't stop after a few user interviews. Weave the information you get from users to iterate, experiment, progress incrementally, and get feedback again.
```
```suggestion
In *Modern Software Engineering*, Farley outlines five behaviors engineers need to become experts in learning: iteration, feedback, incrementalism, experimentation, and empiricism. This shows that the process of learning doesn't stop after a few user interviews. Weave the information you get from users to iterate, experiment, progress incrementally, and get feedback again.
```

jtemperton replied:
weave it in what regard? with that? not especially clear here

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+In *Modern Software Engineering*, Farley outlines 5 behaviors engineers need to become experts in learning: iteration, feedback, incrementalism, experimentation, and empiricism. This shows that the process of learning doesn't stop after a few user interviews. Weave the information you get from users to iterate, experiment, progress incrementally, and get feedback again.
+
+For example, we often work closely with customers to build features and get feedback via Slack. After user interviews, we build solutions, ask customers about them, and iterate. These customers become references for us, their usage represents broader usage. This helps us spend more time developing features that deliver real value.
```
```suggestion
At PostHog we often work closely with customers to build features and get feedback via Slack. After user interviews, we build solutions, ask customers about them, and iterate. These customers become references for us and their usage represents broader usage. This helps us spend more time developing features that deliver real value.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+![Feedback via Slack](https://res.cloudinary.com/dmukukwp6/image/upload/v1713470870/posthog.com/contents/images/newsletter/talk-to-users/feedback.png)
+
+A useful metaphor for engineers is that user interviews are like automated tests. Not doing them saves you time, but leads to pain over the long term. Both enable you to iterate faster by providing rapid feedback on if what you've built is working, even if they aren't directly code you ship.
```
```suggestion
User interviews are like automated tests. Not doing them saves you time, but also leads to pain over the long term. Both enable you to iterate faster by providing rapid feedback on if what you've built is working, even if they aren't code you ship.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,106 @@
+
+A useful metaphor for engineers is that user interviews are like automated tests. Not doing them saves you time, but leads to pain over the long term. Both enable you to iterate faster by providing rapid feedback on if what you've built is working, even if they aren't directly code you ship.
+
+Like your product, your knowledge should be continually improving. Talking to users is one of the best ways to do this. Even if it isn't thought of a tool for software engineers, it should be one for *modern* software engineers.
```
```suggestion
Like your product, your knowledge should be improving all the itme. Talking to users is one of the best ways to do this. Even if it isn't thought of a tool for software engineers, it should be one for modern software engineers.
```

andyvan-ph commented:
FYI @ivanagas @jtemperton, this doesn't work for me atm and needs a bit of a rethink. Some thoughts:

- We spend way too much time on the WHY of talking to users. I'd lost interest halfway through the opening section. I feel like we should just deal with this in the intro so we can get to the actionable bits faster.

- It felt weird to focus on what NOT to talk about before what engineers SHOULD talk about, though the points in that section were useful.

- The 'Iterating beyond user interviews' section felt like a kind of flat, not super engaging ending.

- Some questions I had reading this:
  - How should I prepare for a user interview beforehand?
  - Should should the output of a user interview look like?
  - What's the difference between a good and a bad question?
  - More examples of questions (we have some, but I'd love more)
  - What are the common mistakes n00bs make when talking to users?

On structure, I think it would be if we either:
1) Structure this as a kind of step-by-step guide – i.e. start with finding users, prep, what to do during the interview, output etc.

2) Go for something closer to [10 things we've learned about A/B testing for startups](https://newsletter.posthog.com/p/10-things-weve-learned-about-ab-testing) – i.e. a collection of tips for engineers talking to users.

ivanagas commented:
Ok, rewrote. This is now "8 things engineers should know about talking to users." 

High level feedback wanted, especially on:
- First two sections, strong enough to keep both? Anything I could do to strengthen them? Should I drop them?
- Advice on the last section and ending. I think the point is a natural conclusion, but not fully sure how to word it. 

I'll do another rewrite on the intro.

andyvan-ph reviewed (commented):
Nice one. This is WAAYYY better. High-level notes:

- I do think the first two points are kind of different ways of saying the same thing, and I think the second one says it better. Maybe create an intro out of the first point and start with the second?

- You should refer to your 10x engineer post somewhere, maybe in the intro? It seems very relevant to this topic.

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+In his book *[Modern Software Engineering](https://www.davefarley.net/?p=352)*, Dave Farley writes that mastering software engineering requires becoming an expert in learning. When you prioritize learning, talking to users becomes a critical part of an engineer's toolkit.
+
+To explain further why this is the case, and to help you get started, we wrote this guide based on what we've learned talking to users as engineers ourselves at PostHog.
```
how about something like...

That's why you should be talking to users. But what about how? Here are eight essential tips to make sure you get the most out of those conversations.

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+## 1. Talking to users isn't a waste of time
+
+Many engineers place shipping code above all else. Anything that gets in the way of this is, apparently, a waste of time. This reflects in common metrics for measuring engineering success like lead time or deployment frequency.
```
```suggestion
A lot of engineers place shipping code above all else. Anything that gets in the way of this is, apparently, a waste of time. This reflects in common metrics for measuring engineering success like lead time or deployment frequency.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+Many engineers place shipping code above all else. Anything that gets in the way of this is, apparently, a waste of time. This reflects in common metrics for measuring engineering success like lead time or deployment frequency.
+
+This creates a disregard for shipping the right code: features users value. Studies show this is more common than you might think. For example, [Microsoft](https://ai.stanford.edu/~ronnyk/ExPThinkWeek2009Public.pdf) found that only one-third of ideas tested improved metrics they were designed for.
```
```suggestion
This creates a disregard for shipping the right code and creating features users value. Studies show this is more common than you might think. For example, [Microsoft](https://ai.stanford.edu/~ronnyk/ExPThinkWeek2009Public.pdf) found that only one-third of ideas tested improved metrics they were designed for.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+This creates a disregard for shipping the right code: features users value. Studies show this is more common than you might think. For example, [Microsoft](https://ai.stanford.edu/~ronnyk/ExPThinkWeek2009Public.pdf) found that only one-third of ideas tested improved metrics they were designed for.
+
+Spending more time learning what users want is a fix for this. This helps close the gap between what users value and what you are building.
```
```suggestion
Spending more time learning what users want can solve this problem. And it also helps close the gap between what users value and what you are building.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+## 2. You have an information bottleneck
+
+You might already think you know what users want. Someone spoke to users, turned those insights into tickets, and you to build them. The problem with this is there has been a lot of information lost in the process of going from interviews to tickets.
```
```suggestion
You might already think you know what users want. Someone spoke to users, turned those insights into tickets, and it's on you to build them. Problem is, a lot of information is lost in the process of going from interviews to tickets.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+![Information bottleneck](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/10x-engineers-do-user-interviews/pm-to-engineer.png)
+
+As an engineer, you miss out on the context and depth of information users can provide. The person talking to the user likely doesn't know the options and constraints as well as you do. You miss out on being able to dive deeper on those areas and use that information to close in on a good solution
```
```suggestion
As an engineer, you miss out on the context and depth of information users can provide. The person talking to the user likely doesn't know the options and constraints as well as you do. You miss out on being able to dive deeper on those areas and use that information to find a good solution
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+As an engineer, you miss out on the context and depth of information users can provide. The person talking to the user likely doesn't know the options and constraints as well as you do. You miss out on being able to dive deeper on those areas and use that information to close in on a good solution
+
+Think of user interviews like automated tests. Not doing them saves you time, but also leads to pain over the long term. Both enable you to iterate faster by providing rapid feedback on if what you've built is working, even if they aren't code you ship.
```
```suggestion
Think of user interviews like automated tests. Not doing them saves you time, but also leads to pain over the long term. Both enable you to iterate faster by providing rapid feedback on whether what you've built is working, even if they aren't code you ship.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+Think of user interviews like automated tests. Not doing them saves you time, but also leads to pain over the long term. Both enable you to iterate faster by providing rapid feedback on if what you've built is working, even if they aren't code you ship.
+
+## 3. Preparing to talk to users
```
this applies to all the crossheads, but if the title is '8 things engineers should know about talking to users' then the crossheads need to be those things. so more like

3. You've got to prepare
4. Find the right users to talk to

just little tweaks to how the crossheads are phrased

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+To make the most out of talking to users, you should know: 
+
+1. **Who are your users.** Read "[Defining our ICP is the most important thing we ever did](https://newsletter.posthog.com/p/defining-our-icp-is-the-most-important)" if you don't know.
```
```suggestion
1. **Who are your users.** Read about how we [defined our ICP](https://newsletter.posthog.com/p/defining-our-icp-is-the-most-important) if you don't know the answer to this.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+1. **Who are your users.** Read "[Defining our ICP is the most important thing we ever did](https://newsletter.posthog.com/p/defining-our-icp-is-the-most-important)" if you don't know.
+
+2. **How they are using your product (or competitors).** Watch [session replays](https://posthog.com/session-replay) and use analytics to figure out popular features. Analyze competitors for their feature sets and use cases. 
```
can we lose the parenthesis here?

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+3. **What you want to build next.** Understand [the significant problems you could work on](https://posthog.com/founders/product-market-fit-game#level-1---find-a-significant-problem-to-work-on) and create a potential roadmap to solve,  but be open to completely changing it.
+
+Less theoretically, it's also worth it to set up tools that help you automate the process. These could include PostHog's [user interview survey](https://posthog.com/tutorials/feedback-interviews-site-apps#using-surveys-to-book-user-interviews) and Calendly for scheduling, your video conferencing tool of choice, and an AI notetaker like [Superpowered](https://superpowered.me/).
```
```suggestion
You should also set up tools that help you automate the process. These could include PostHog's [user interview survey](https://posthog.com/tutorials/feedback-interviews-site-apps#using-surveys-to-book-user-interviews) and Calendly for scheduling, your video conferencing tool of choice, and an AI notetaker like [Superpowered](https://superpowered.me/).
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+## 4. Finding the right users to talk to
+
+Just because we convinced you that talking to users isn't a waste of time doesn't mean we want to waste your time doing so. Lucky for you, you don't need a lot of users. You don't need to find a statistically significant sample. A few user interviews can be all you need to clarify the next increment of progress.
```
```suggestion
You don't need to talk to a huge number of users, nor do you need a statistically significant sample. A few user interviews can be all you need to clarify the next increment of progress.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+1. **Yourself or your product.** If they are users, they probably know what your product is and what it does.
+
+2. **Your solution or idea.** Resist the urge to explain your thinking. You will bias their answers. Focus on their problems instead. Asking "Would you use this?"  leads to failure.
```
```suggestion
2. **Your solution or idea.** Resist the urge to explain your thinking. You will bias their answers. Focus on their problems instead. Asking "Would you use this?" leads to failure.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+4. **Macroanalysis**. Focus on an individual's specific use case rather than a company or industry one. Outside research can fill this if necessary.
+
+Any focus other than their problems is a bad one. 
```
this sentence feels a bit lost?

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+![User interview doc](https://res.cloudinary.com/dmukukwp6/image/upload/v1713832564/posthog.com/contents/images/newsletter/talk-to-users/interview-doc.png)
+
+The second thing to do after the interview is to use it to take action. The great part about being an engineer is that you can just ship small changes yourself. For example, our team often immediately makes small UX changes to clarify a user confusion.
```
```suggestion
The second thing to do after the interview is to take action. The great part about being an engineer is that you can just ship small changes yourself. For example, our team often makes small UX changes to clarify a user confusion.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+## 8. Talking to users doesn't stop at user interviews
+
+Your users are still your users after you talk with them. This means you should still try to build valuable things and get feedback from them after you talk with them.
```
```suggestion
Your users are still your users after you talk to them. This means you should still try to build valuable things and get feedback from them after your conversation.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+Your users are still your users after you talk with them. This means you should still try to build valuable things and get feedback from them after you talk with them.
+
+At PostHog, we often work closely with customers to build features and get feedback via Slack. After user interviews, we build solutions, ask customers about them, roll them out via [feature flags](https://posthog.com/feature-flags), and iterate. These customers become references for us and their usage represents broader usage.
```
```suggestion
At PostHog, we work closely with customers to build features and get feedback via Slack. After user interviews, we build solutions, ask customers about them, roll them out via [feature flags](https://posthog.com/feature-flags), and iterate. These customers become references for us and their usage represents broader usage.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+
+![Feedback via Slack](https://res.cloudinary.com/dmukukwp6/image/upload/v1713470870/posthog.com/contents/images/newsletter/talk-to-users/feedback.png)
+
+By getting more feedback from users, we are better able to ship features users actually want. In the long run, this is a repeatable method to build the best possible product.
```
```suggestion
By getting more feedback from users, we are better able to ship features they actually want. In the long run, this is a repeatable method to build the best possible product.
```

jtemperton commented on contents/newsletter/talk-to-users.md:
```diff
@@ -0,0 +1,141 @@
+  - Blog
+---
+
+Talk to users? Isn't that a job for founders, product managers, and salespeople? Why would an engineer ever want or need to do it?
```
```suggestion
Talking to users? Isn't that a job for founders, product managers, and salespeople? Why would an engineer ever want or need to do it?
```

jtemperton reviewed (approved):
Thumbs up from @andyvan-ph on the changes from yesterday and I've done another edit on the new structure this morning. We'll still run this one this week, so if you could wrap up edits ASAP today that'd be great. Thanks!
